docs/en/query_language/misc.md

```diff
@@ -32,15 +32,16 @@ The query response contains the `result` column with a single row. The row has a
 - 0 - The data in the table is corrupted.
 - 1 - The data maintains integrity.
 
-The `CHECK TABLE` query is only supported for the following table engines:
+The `CHECK TABLE` query works for the following table engines:
 
 - [Log](../operations/table_engines/log.md)
 - [TinyLog](../operations/table_engines/tinylog.md)
 - [StripeLog](../operations/table_engines/stripelog.md)
+- [MergeTree](../operations/table_engines/mergetree.md)
```

Comment:
It's worth to explicitly mention whether it's the whole MergeTree family or only the basic one.